src/components/Navbar/NavBar.jsx

```diff
@@ -0,0 +1,39 @@
+import { HamburgerIcon } from '@chakra-ui/icons'
+import { Input } from '@chakra-ui/input'
+import { Box, Flex, Text } from '@chakra-ui/layout'
+import { Switch } from '@chakra-ui/switch'
+import { CgProfile } from 'react-icons/cg';
+import React from 'react'
+import SideMenu from '../SideMenu/SideMenu';
+import { useDisclosure, Link } from "@chakra-ui/react";
+import NextLink from "next/link";
+import Title from '../Title/Title';
```

Comment:
```suggestion
import SideMenu from '../SideMenu/SideMenu'
import { useDisclosure, Link } from "@chakra-ui/react"
import NextLink from "next/link"
import Title from '../Title/Title'
```

I used to work with python a lot and I like how javascript looks without semicolons but most of the people I have work with use them and sometimes there are formatters that add stuff like semicolons, changes indentation to two spaces, etc that run when you commit to the repo. This is why I ended up using semicolons even do I don't like them a lot. Sharing some of my experiences, I hope it is useful in someway